Data/test.lua

```diff
@@ -0,0 +1,9646 @@
+SetSource({
```

Comment:
It might be interesting to leave a comment or something about the LST file it came from (in case we need to regenerate something, or something looks weird).  The name/line number/ maybe even commit hash, would help a lot. :-)